docs/docs/backend/dashboard.md

```
@@ -0,0 +1,2043 @@
+
+## ConfigDashboard Objects
+
+```python
+class ConfigDashboard(ConfigAPIClient, ConfigAPI)
+```
+
+Class gathers all configuration information.
+
+<a id="configuration.ConfigDashboard.__init__"></a>
+
+#### \_\_init\_\_
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)